modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -2495,4 +2497,31 @@ public int tableId() {
             return tableId;
         }
     }
+
+    private void cleanUpResourcesForDroppedTablesOnRecoveryBusy() {
```

Comment:
Do we have an integration test that covers this code? Like: create a table, stop table manager, update low watermark, start table manager, check that the tables are cleaned up.

Reply:
Well damn. Fair enough, I’ll write something like this.

Reply:
Add checks